conans/model/build_info.py

```diff
@@ -207,8 +208,14 @@ def loads(text):
         return result
 
     def update(self, dep_cpp_info, conan_ref):
-        assert isinstance(dep_cpp_info, (CppInfo, DepsCppInfo))
         self._dependencies[conan_ref.name] = dep_cpp_info
         super(DepsCppInfo, self).update(dep_cpp_info)
         for config, cpp_info in dep_cpp_info.configs.items():
             self.configs.setdefault(config, _BaseDepsCppInfo()).update(cpp_info)
+
+    def update_deps_cpp_info(self, dep_cpp_info):
+        assert isinstance(dep_cpp_info, DepsCppInfo)
+        for pkg, cpp_info in dep_cpp_info.dependencies:
+            from collections import namedtuple
+            ref = namedtuple('ref', "name")(pkg)
```

Comment:
And filled with which values? Pkg/None@None/None?
The other possibility is to change the function so it directly accepts just the package name, and not the conan_ref. I initially tried, but stepped back because it was a bit dirty somewhere.